iocage/lib/JailConfig.py

```diff
@@ -0,0 +1,612 @@
+import re
+
+import iocage.lib.JailConfigJSON
+import iocage.lib.JailConfigInterfaces
+import iocage.lib.JailConfigAddresses
+import iocage.lib.JailConfigResolver
+import iocage.lib.JailConfigFstab
+import iocage.lib.JailConfigLegacy
+import iocage.lib.JailConfigZFS
+import iocage.lib.helpers
+
+
+class JailConfig():
+
+    def __init__(self, data={}, jail=None, logger=None, new=False):
+
+        iocage.lib.helpers.init_logger(self, logger)
+
+        object.__setattr__(self, 'data', {})
+        object.__setattr__(self, 'special_properties', {})
+        object.__setattr__(self, 'legacy', False)
```

Comment:
That looks much more straightforward and allow properties with characters not allowed in a python attribute name.